### PR TITLE
Updated some language files

### LINKED ISSUE #848: Sync translations
- Sync translations between CrowdIn and Cachet source.
- Remove any translations that are not >= 80% translated.

### PR DESCRIPTION
**Pull request by jbrooksuk**
Closes #848

### CODE REVIEW

File: resources/lang/de/cachet.php [hunk 1]
```diff
@@ -14,82 +14,84 @@
...
+                'text'           => 'Ein neuer Vorfall wurde auf :app_name berichtet.\\nDanke, :app_name',
+                'html-preheader' => 'Ein neuer Vorfall wurde über :app_name berichtet.',
+                'html'           => '<p>Ein neuer Vorfall wurde über :app_name berichtet.</p><p>Danke, :app_name</p>',
             ],
         ],
     ],
 
     // Other
-    'powered_by'      => ':app Status-Seite via <a href="https://cachethq.io">Cachet</a>.',
+    'powered_by'      => ':app Statusseite betrieben mit <a href="https://cachethq.github.io">Cachet</a>.',
```
Comment (ApfelUser) [inline resources/lang/de/cachet.php]: I think this is wrong.

Comment (jbrooksuk) [inline resources/lang/de/cachet.php]: Cheers, I'll download again shortly.

File: resources/lang/id/pagination.php [hunk 2]
```diff
@@ -3,7 +3,7 @@
 /*
  * This file is part of Cachet.
  *
- * (c) Alt Three Services Limited
+ * (c) Cachet HQ <support@cachethq.io>
```
Comment (GrahamCampbell) [inline resources/lang/id/pagination.php]: wrong

File: resources/lang/id/reminders.php [hunk 3]
```diff
@@ -1,14 +1,5 @@
 <?php
 
-/*
- * This file is part of Cachet.
- *
- * (c) Alt Three Services Limited
```
Comment (GrahamCampbell) [inline resources/lang/id/reminders.php]: wrong

Comment (GrahamCampbell) [inline resources/lang/id/reminders.php]: Also, the file permissions are wrong.

File: resources/lang/id/pagination.php (hunk 2 above)
Comment (jbrooksuk) [inline resources/lang/id/pagination.php]: Should StyleCI be fixing this? CrowdIn generated it incorrectly.

Comment (GrahamCampbell) [inline resources/lang/id/pagination.php]: > Should StyleCI be fixing this?

It would if we actually had the header config added...

Comment (GrahamCampbell) [inline resources/lang/id/pagination.php]: Added it now.

**Comment by GrahamCampbell** [comment]
Looks like this branch needs rebasing too.

File: resources/lang/id/pagination.php (hunk 2 above)
Comment (jbrooksuk) [inline resources/lang/id/pagination.php]: Oh haha!

File: resources/lang/de/cachet.php (hunk 1 above)
Comment (jbrooksuk) [inline resources/lang/de/cachet.php]: Thanks @ApfelUser - updated!

**Comment by jbrooksuk** [comment]
Squashed, rebased and good to go!

**Pull request merged**